docs/shipping/Code/nestjs.md

```diff
@@ -327,19 +327,24 @@ logzio-k8s-telemetry logzio-helm/logzio-k8s-telemetry
 {@include: ../../_include/tracing-shipping/replace-tracing-token.html}
 `<<LOGZIO_ACCOUNT_REGION_CODE>>` - Your Logz.io account region code. [Available regions](https://docs.logz.io/docs/user-guide/admin/hosting-regions/account-region/#available-regions).
 
-#### Define the logzio-k8s-telemetry service dns
+#### Define the service DNS
+
+You'll need the following service DNS:
+
+`http://<<CHART-NAME>>-otel-collector.<<NAMESPACE>>.svc.cluster.local:<<PORT>>/`.
+
+Replace `<<CHART-NAME>>` with the relevant service you're using (`logzio-k8s-telemetry-otel-collector`, `logzio-k8s-telemetry`).
```

Comment:
The k8s template for service DNS is:
```
http://<<SERVICE_NAME>>.<< NAMESPACE>>.svc.cluster.local
```

Since `<<CHART-NAME>>` is `logzio-k8s-telemetry`, I think maybe we can either:

- Remove `logzio-k8s-telemetry-otel-collector` from the example since the template already includes `-otel-collector`
- Update the template to not hardcode `-otel-collector`

What do you think?

Reply:
I removed it from the example, but I kept the `logzio-k8s-telemetry`